### ISSUE TITLE
socket.io version ?

### ISSUE THREAD

**Issue by ugurberkecan** [ChainSafe/lazy-minting-server#3]
Could you provide, socket.io version of it ?

**Comment by harsh-na** [comment]
4.5.3 is the latest socket.io version.

**Comment by RyRy79261** [comment]
@ugurberkecan @harsh-na 

Haven't considered it, haven't implemented sockets before

What would the requirements be? I can ask the team to work on it if I can flesh out what's needed

**Comment by RyRy79261** [comment]
For reference though, this code base was meant to be a minimum implementation of the most simple pseudocode 

This was more to demonstrate the steps needed, rather than provide a solved solution

For example the Nonce and actually issuing the voucher, the usage and requirements to get the voucher is very game specific, could be resolving the status of matches, it could be the result of farm crops in a game, it could be based on how long the gamer has been playing, etc

So what ever steps a game might need between the user being eligible for the nft, would be up to the game dev, and delivered via their preferred stack

**Comment by harsh-na** [comment]
@RyRy79261 
I could not get the problem you had described in above comments, the only thing i got is the versioning of socket.io being asked.

**Comment by RyRy79261** [comment]
@harsh-na The scenario I described is saying that this is not a production level repo and is meant to be a absolute minimum implementation of having a server sign a voucher. 

If you are building a game, you will need bespoke logic, rules, etc between issuing users nfts and handling their requests.

tl;dr: this is a reference repo
